test/project_types/script/layers/infrastructure/languages/task_runner_test.rb

```diff
@@ -26,4 +28,62 @@
       end
     end
   end
+
+  describe "check_tool_version!" do
+    let(:tool) { "npm" }
+    let(:tool_version) { EXACT_NPM_VERSION }
+
+    subject { task_runner.send(:check_tool_version!, tool, tool_version) }
```

Comment:
Just curious, why do you use `task_runner.send` instead of `task_runner.check_tool_version!` directly?